apps/prairielearn/elements/pl-external-grader-results/pl-external-grader-results.mustache

```diff
@@ -50,8 +50,8 @@
           <ul class="list-group list-group-flush">
               {{#images}}
                   <li class="list-group-item">
-                      <div><strong>{{#label}}{{label}}{{/label}}{{^label}}Figure{{/label}}</strong></div>
-                      <img src="{{#url}}{{url}}{{/url}}{{^url}}{{.}}{{/url}}"/>
+                      <div><strong>{{label}}{{^label}}Figure{{/label}}</strong></div>
```

Comment:
Nit: don't let this stand in the way of merging, but we could use a `<figure>` element to associate the label (caption) with the image for improved accessibility: https://getbootstrap.com/docs/5.3/content/figures/

Reply:
Done in 9b94238f8. I didn't use the Bootstrap `figure-caption` class as I wanted to keep the same label formatting used in the rest of the element (the merits of that can be discussed in a separate PR).